web-src/src/main-app/components/schedule/SchedulePanel.vue

```diff
@@ -5,62 +5,79 @@
       <div class="schedule-type-panel">
         <p class="schedule-type-field">
           <label>
-            <input :checked="oneTimeSchedule" class="with-gap" name="schedule-type"
-                   type="radio"
-                   @click="oneTimeSchedule = true"/>
+            <input :checked="oneTimeSchedule" class="with-gap" name="schedule-type" type="radio" @click="oneTimeSchedule = true" />
```

Comment:
Could you revert changes in the existing lines please, which you didn't modify here? I really prefer my previous formatting (with new lines), rather than long lines, which are much harder to read.

Reply:
Changes should be reverted.

Reply:
Thanks!